zorka-core-test/src/main/resources/scripts/wildcard-example.bsh

```diff
@@ -0,0 +1,58 @@
+// This is sample script intended to help understanding how to implement arbitrary method instrumentation.
+
+// To run:
+//  1. update zorka.properties to add it to list of loaded scripts, i.e. scripts = jvm.bsh, wildcard-example.bsh
+//  2. update zorka.properties to set zorka.application = Caculator
```

Comment:
I'd like to point out that this is copy pasted from calculator.bsh, of which I am not the author haha